cfme/services/catalogs/orchestration_template.py

```diff
@@ -237,7 +239,7 @@ class AddDialog(CFMENavigateStep):
     VIEW = AddDialogView
 
     def step(self):
-        self.view.configuration.item_select('Create Service Dialog from Orchestration Template')
+        cfg_btn("Create Service Dialog from Orchestration Template")
```

Comment:
@jteehan What exactly was the error it was ending with? I just checked if it is a Widgetastic problem, but the dropdown works fine with this string both on 5.6 and 5.7 in that location. Maybe the navigation did something wrong and then the configuration button did not match?

With regards of initializing the self.view.configuration ... well that is the point of the whole design, that the things just get lazily regenerated per view to avoid any possible validity problems :)